### ISSUE TITLE
Migrating multiple streams

### ISSUE THREAD

**Issue by vivshankar** [rtcTo/rtc2gitcli#42]
Hi,

This is a great tool! Thanks for the work you have done here.

I am migrating a fairly big product that contains several streams - several maintenance and one master. Do you have any recommendations for migrating multiple streams?

**Comment by WtfJoke** [comment]
Hi vivshankar

We had a similar situation. We made a plan, which stream goes in which git repo. 

For each planned git repo, we migrated all the planned streams in there.
Then we distributed the git repo and proceeded with the next streams/repos.

**Comment by vivshankar** [comment]
Thanks. So you ended up releasing multiple github repos each mapping to one RTC stream? That's what I am doing now, because I can see no way of building a linked commit history. I can see that, if you have a solid set of snapshots/baselines, this could be possible, in theory. 

You would migrate the master stream and tag each baseline. Then branch out of each baseline and migrate the left over changesets from each maintenance stream. Of course, you lose the commit linking at that point, but it is better than nothing. Unfortunately, my RTC repo doesn't look that beautiful :-D

**Comment by WtfJoke** [comment]
Yes, we released multiple git repos each mapping to one or **more** RTC Streams. Usually all the streams which belongs together we migrated in one git repo. 

We grouped up to about 5 Streams in one repo. Sometimes one stream = one repo (if it contains a lot of binary files for example - allthough we tried to filter them out)

I think the baselines will be converted in tags, but not 100% sure.
We migrated also the maintancestreams into the same git repo but in a separate branch if I remember correctly.
